src/Service.GraphQLBuilder/Mutations/CreateMutationBuilder.cs

```diff
@@ -143,7 +194,8 @@ private static InputValueDefinitionNode GenerateSimpleInputType(NameNode name, F
         /// <param name="definitions">All named GraphQL types from the schema (objects, enums, etc.) for referencing.</param>
         /// <param name="field">Field that the input type is being generated for.</param>
         /// <param name="typeName">Name of the input type in the dictionary.</param>
-        /// <param name="otdn">The GraphQL object type to create the input type for.</param>
+        /// <param name="baseObjectTypeName">Name of the underlying object type of the field for which the input type is to be created.</param>
```

Comment:
In case of relationships with M:N cardinality, `typeName` = type name of linking object, else `typeName` = type name of target entity. While `baseObjectTypeName` always stores the object type name of the target entity. I will add this as param summary.